static/css/section/_cloud.scss

```diff
@@ -43,6 +43,22 @@
 /*	@section cloud
 -------------------------------------------------------------- */
 
+%icon {
+  margin-left: 0;
+  background-size: 20px 20px;
+  box-sizing: border-box;
+  background-position: 0 5px;
+  background-repeat: no-repeat;
+  display: block;
+  float: left;
+  padding: 6px 20px 6px 24px;
+  width: 100%;
+  font-size: .92857143em;
```

Comment:
What font-size are you going for here? Could you use `rem` to be easier to read?